libcodechecker/analyze/analyzer.py

```diff
@@ -129,7 +129,8 @@ def perform_analysis(args, context, actions, metadata):
     if ctu_analyze or (not ctu_analyze and not ctu_collect):
         analysis_manager.start_workers(actions, context, config_map,
                                        args.jobs, args.output_path,
-                                       __get_skip_handler(args), metadata)
+                                       __get_skip_handler(args),
+                                       metadata, args)
```

Comment:
Propagating the `args` here makes it really redundant, as `args.jobs` and `args.output_path` is also propagated.

It is also isn't the best approach to propagate a magic dynamic configuration container (such as `args` in this case) so deep and so much across thread and subprocess boundaries in the code.
It makes for a much better documented behaviour if we only passed the actual boolean here, and used it later on.